components/crud-web-apps/jupyter/manifests/base/configs/spawner_ui_config.yaml

```diff
@@ -25,14 +25,14 @@ spawnerFormDefaults:
       - public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter-pytorch-cuda-full:master-1831e436
       - public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter-tensorflow-full:master-1831e436
       - public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/jupyter-tensorflow-cuda-full:master-1831e436
-  imageVSCode:
-    # The container Image for the user's VS-Code Server
+  imageGroupOne:
+    # The container Image for the user's Group One Server
     value: public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/codeserver-python:master-1831e436
     # The list of available standard container Images
     options:
       - public.ecr.aws/j1r0q0g6/notebooks/notebook-servers/codeserver-python:master-1831e436
-  imageRStudio:
-    # The container Image for the user's RStudio Server
+  imageGroupTwo:
+    # The container Image for the user's Group Two Server
```

Comment:
same as the comment above to add a comment for explaining the logic we apply to images in this group